docs/markdown.rst

```diff
@@ -2,56 +2,106 @@ markdown
 ========
 
 .. dfhack-tool::
-    :summary: Exports the text you see on the screen for posting online.
-    :tags: unavailable
-
-This tool saves a copy of a text screen, formatted in markdown, for posting to
-Reddit (among other places). See `forum-dwarves` if you want to export BBCode
-for posting to the Bay 12 forums.
-
-This script will attempt to read the current screen, and if it is a text
-viewscreen (such as the dwarf 'thoughts' screen or an item 'description') then
-append a marked-down version of this text to the output file. Previous entries
-in the file are not overwritten, so you may use the ``markdown`` command
-multiple times to create a single document containing the text from multiple
-screens, like thoughts from several dwarves or descriptions from multiple
-artifacts.
-
-The screens which have been tested and known to function properly with this
-script are:
-
-#. dwarf/unit 'thoughts' screen
-#. item/art 'description' screen
-#. individual 'historical item/figure' screens
-#. manual pages
-#. announcements screen
-#. combat reports screen
-#. latest news (when meeting with liaison)
-
-There may be other screens to which the script applies. It should be safe to
-attempt running the script with any screen active. An error message will inform
-you when the selected screen is not appropriate for this script.
+    :summary: Save descriptions of selected units/items in markdown format (e.g., for Reddit).
+    :tags: fort | inspection | units
+
+Saves descriptions of selected units or items to a markdown file.
+
+
+For units, the script retrieves:
+#. Name, race, age, profession
+#. Description from the Unit/Health/Description screen
+#. Traits from the Unit/Personality/Traits screen
+The script works for all units with text in the Description &/or Personality/Traits tabs, 
+such as dwarves, dogs, elves, goblins, and beasts.
+
+For items, it retrieves:
+#. Decorated name (e.g., "☼«☼Chalk Statue of Dakas☼»☼")
+#. Full description from the item's view sheet
+The script works for most items with in-game descriptions and names, including those in storage,
+on the ground, installed as a building, or worn/carried by units.
+
+The script appends a markdown-formatted version of the text to a target file 
+for easy sharing, e.g., on Reddit.
+
+By default, entries are appended, not overwritten, allowing the ``markdown`` command 
+to compile descriptions of multiple items & units in a single document.
+
+By default, data is stored in markdown_{YourWorldName}_export.md.
+
+See `forum-dwarves` for BBCode export (e.g., for the Bay12 Forums).
```

Comment:
Unless you're planning on restoring `forum-dwaves` (or folding it into this tool), I'd take this reference out.